flink-formats/flink-avro-confluent-registry/src/main/java/org/apache/flink/formats/avro/registry/confluent/ConfluentRegistryAvroDeserializationSchema.java

```diff
@@ -84,14 +86,50 @@ private ConfluentRegistryAvroDeserializationSchema(
      */
     public static ConfluentRegistryAvroDeserializationSchema<GenericRecord> forGeneric(
             Schema schema, String url, int identityMapCapacity) {
+        return forGeneric(schema, url, identityMapCapacity, null);
+    }
+
+    /**
+     * Creates {@link ConfluentRegistryAvroDeserializationSchema} that produces {@link
+     * GenericRecord} using provided reader schema and looks up writer schema in Confluent Schema
+     * Registry.
```

Comment:
nit (and same below):

```suggestion
     * Creates {@link ConfluentRegistryAvroDeserializationSchema} that produces {@link
     * GenericRecord} using the provided reader schema and looks up the writer schema in the Confluent Schema
     * Registry.
```